src/LocaliseInterface.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Part of the Joomla Framework Language Package
+ *
+ * @copyright  Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Language;
+
+/**
+ * Joomla Framework Language Interface
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+interface LocaliseInterface
```

Comment:
There isn't anything special about the Localise object that warrants an interface IMO.  I think I'd suggest just having `AbstractLocalise` be an abstract classes and language specific implementations extend that as needed.  For some, it may be an empty class.

And like I mentioned elsewhere, this moves the Language object in a direction where the localization object is a hard requirement whereas previously you could work without it.  We need to be absolutely certain we want to add a layer of complexity, no matter how small it is, to the class.